### PR TITLE
feat/users management search bar

### PR DESCRIPTION
**Pull request by ItayBenMoshe**
[434](https://dev.azure.com/spectrumFactory/CoronaI/_workitems/edit/434/)

### CODE REVIEW

File: client/src/components/App/Content/UsersManagement/UsersManagementStyles.ts [hunk 1]
```diff
@@ -33,10 +33,10 @@ const useStyles = makeStyles((theme: Theme) => ({
     active: {},
     filters: {
         flip: false,
+        width: '90vw',
         margin: theme.spacing(2),
         display: 'flex',
-        justifyContent: 'flex-end',
-        marginLeft: '10vw'
+        justifyContent: 'space-between',
```
Comment (y-golde) [inline client/src/components/App/Content/UsersManagement/UsersManagementStyles.ts:39]: you can use justify="space-between" the Grid instead, Display: flex is redundant and you can use spacing={2} on Grid as well

**Review by y-golde — CHANGES_REQUESTED** [review]

File: client/src/components/App/Content/UsersManagement/UsersManagementStyles.ts (hunk 1 above)
Comment (y-golde) [reply client/src/components/App/Content/UsersManagement/UsersManagementStyles.ts:39]: meaning `<Grid justify="space-between" spacing={2} />`

Comment (ItayBenMoshe) [reply client/src/components/App/Content/UsersManagement/UsersManagementStyles.ts]: fixed

File: client/src/components/App/Content/UsersManagement/UsersFilter/SearchBar.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    }
+
+    const onClearClick = () => {
+        setSearchQuery('');
+        !isQueryValid && setIsQueryValid(true);
+        onClick(''); 
+    }
+
+    return (
+        <TextField
```
Comment (RoeiRom) [inline client/src/components/App/Content/UsersManagement/UsersFilter/SearchBar.tsx:37]: why not using AlphanumericTextField?

**Review by y-golde — APPROVED** [review]

**Review by RoeiRom — CHANGES_REQUESTED** [review]

File: client/src/components/App/Content/UsersManagement/UsersFilter/SearchBar.tsx (hunk 2 above)
Comment (ItayBenMoshe) [reply client/src/components/App/Content/UsersManagement/UsersFilter/SearchBar.tsx:37]: I'm pretty sure it works only under form

**Review by Noam6421 — APPROVED** [review]

File: client/src/components/App/Content/UsersManagement/UsersFilter/SearchBar.tsx (hunk 2 above)
Comment (ItayBenMoshe) [reply client/src/components/App/Content/UsersManagement/UsersFilter/SearchBar.tsx:37]: it use useFormContext which i don't have

**Review by RoeiRom — APPROVED** [review]

**Pull request merged**